docs/config.rst

```diff
@@ -380,8 +380,7 @@ Implementation
 
         @Override
         public Raven createRavenInstance(Dsn dsn) {
-            Raven raven = new Raven();
-            raven.setConnection(createConnection(dsn));
```

Comment:
Should we move `setConnection` to protected scope or remove it completely at this point, even though it wasn't explicitly deprectaed? I only see it being used in tests, and can't think of any justifiable reason you'd want to replace the connection without instantiating a new `Raven` instance.